.gitignore

```diff
@@ -0,0 +1,4 @@
+# Ignore node_modules folder
+node_modules/
+# Ignore text.md file
+text.md
```

Comment:
This file was supposed to be called test.md😀 but as long as you understand how to gitignore files. that is the main point 👍.